code/processes/tailer.q

```diff
@@ -50,9 +51,10 @@ getprocs:{[x;y]
 \d .
 
 /- initialise datastripe
-.lg.o[`dsinit;"initialising datastripe"];
-initdatastripe[];
-
+if[.ds.datastripe;
+  .lg.o[`dsinit;"datastripe on: initialising datastripe"];
+  initdatastripe[]];
 
 /- create HDB sym file and taildir symlink
-.ds.symlink[];
+if[.ds.datastripe;
+  .ds.symlink[]];
```

Comment:
Shouldn't be needed as tailer will only be on if .ds.datastripe is enabled in the STP.

Reply:
No worries, will streamline this by the end of the day